src/utils/mail.ts

```diff
@@ -0,0 +1,28 @@
+import nodemailer from "nodemailer";
+
+const transporter = nodemailer.createTransport({
+  host: process.env.SMTP_HOST,
+  port: 587,
+  secure: true,
+  auth: {
+    user: process.env.SMTP_USERNAME,
+    pass: process.env.SMTP_PASSWORD,
+  },
+});
+
+export async function sendMail(
+  to: string,
+  subject: string,
+  text: string,
+  html: string
+) {
+  const mailOptions = {
+    from: process.env.SMTP_USERNAME,
```

Comment:
to bym raczej przerzucił do transportera

```ts
const transporter = nodemailer.createTransport(
  {
    host: env.SMTP_HOST,
    port: 587,
    secure: true,
    auth: {
      user: env.SMTP_USERNAME,
      pass: env.SMTP_PASSWORD,
    },
  },
  {
    from: env.SMTP_USERNAME,
  }
);
```

plus raczej nie chcemy mieć czystego maila jako nazwe, dodałbym zmienną środowiskową SENDER_NAME i zrobił coś takiego

```ts
const transporter = nodemailer.createTransport(
  {
    host: env.SMTP_HOST,
    port: 587,
    secure: true,
    auth: {
      user: env.SMTP_USERNAME,
      pass: env.SMTP_PASSWORD,
    },
  },
  {
    from: `${env.SENDER_NAME} ${env.SMTP_USERNAME}`,
  }
);
```